.github/workflows/prerelease-exit.yml

```diff
@@ -0,0 +1,43 @@
+name: Exit Prerelease Mode
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Exit prerelease mode on release branch"
+        type: string
+        default: "main"
+        required: true
+
+jobs:
+  exit_prerelease:
+    name: Changesets Exit Prerelease
+    runs-on: ubuntu-latest
+    # Allow GITHUB_TOKEN to have write permissions
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout Repo
+        with:
+          submodules: recursive
+
+      - name: "Setup"
+        uses: ./.github/actions/setup
+        
+      - name: Generate gas reports
+        run: pnpm gas-report
+
+      - name: Outdated files, run `pnpm gas-report` and commit them
+        uses: ./.github/actions/require-empty-diff
+
+      - name: Remove pre.json
+        run: npx rimraf .changeset/pre.json
```

Comment:
what's the difference between removing the `pre.json` file to running `pnpm changeset pre exit` (from https://github.com/changesets/changesets/blob/main/docs/prereleases.md)?

Reply:
this is a bug I forgot to change this from the original action I copy pastad

Reply:
We should use the action the one I copy pastad was not clean and doing this for pretty hacky reasons